.circleci/config.yml

```diff
@@ -256,7 +202,6 @@ jobs:
             export VERSION_TO_DEPLOY="$(cat /tmp/build-info/version-to-deploy.txt)" && \
             ssh -t ciuser@pfs-management-bastion-1.uksouth.cloudapp.azure.com ssh -t ciuser@hub-wayland-prod.pfs-management.com "sudo /etc/docker-decomposed-secrets.sh -v ${VERSION_TO_DEPLOY}" && \
```

Comment:
Don't these want to go to? We're not in Azure in prod either? Or do you just want to keep prod up-to-date for now?

Reply:
Good point, this PR is a little out of date and we can get rid of even more now :+1: